### PR TITLE
It's high noon...

### PR DESCRIPTION
**Pull request by silicons**
THIS IS A TG PORT BTW love you @AnturK 

## About The Pull Request

Adds dueling pistols to the game, with hugbox versions for when you want to just duel without disintegrating someone's head. 
Also, the "hugbox" versions are powergamer proof :)

## Why It's Good For The Game

Dueling, 'nuff said.

Added them to cargo, lethal ones for contraband, and the disintegration ones for emag.

Holodeck has it on thunderdome.

## Changelog
:cl: AnturK ported by kevinz000
add: Dueling pistols have been added.
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs/misc.dm [hunk 1]
```diff
@@ -214,6 +214,34 @@
...
+	contains = list(/obj/item/storage/lockbox/dueling/hugbox/stamina,
+	/obj/item/storage/lockbox/dueling/hugbox/stamina,
+	/obj/item/storage/lockbox/dueling/hugbox/stamina,
+	/obj/item/storage/lockbox/dueling/hugbox/stamina,
+	/obj/item/storage/lockbox/dueling/hugbox/stamina)
+	crate_name = "dueling pistols"
+
+/datum/supply_pack/misc/dueling_lethal
+	name = "Lethal Dueling Pistols"
+	desc = "Resolve all your quarrels with some nonlethal fun."
```
Comment (necromanceranne) [inline code/modules/cargo/packs/misc.dm]: "Resolve all your quarrels with some lethal fun." is what you meant ;)

**Comment by Xantholne** [comment]
An excellent edition to the red dead redemption 2 in space pr i made a while back

**Comment by Linzolle** [comment]
wait why is the branch called duke_moxley lmfao

**Review by Ghommie — APPROVED** [review]

**Comment by ghost** [comment]
epic

**Pull request merged**